plugins/custom-codecs/src/main/java/org/opensearch/index/codec/customcodecs/ZstdCodec.java

```diff
@@ -49,4 +59,9 @@ public String toString() {
     public boolean supports(Setting<?> setting) {
         return setting.equals(EngineConfig.INDEX_CODEC_COMPRESSION_LEVEL_SETTING);
     }
+
+    @Override
+    public Set<String> aliases() {
+        return Set.of("zstd");
```

Comment:
nit: can the aliases also be made as part of Mode itself?

```suggestion
        return Mode.ZSTD.aliases();
```